src/plugins/util/web.js

```diff
@@ -146,35 +155,66 @@ const web = {
     if (req._datadog.middleware.length === 0) return req._datadog.span || null
 
     return req._datadog.middleware.slice(-1)[0]
+  },
+
+  // Extract the parent span from
+  extractSpan (tracer, name, headers) {
```

Comment:
This is misleading since it doesn't just extract the parent but also creates a new span. The comment also doesn't seem to be complete which doesn't help with clarity.

Reply:
Good catch, I've changed it to `startChildSpan(...)` for now, but I'm open to suggestions!